adserver/templates/adserver/base.html

```diff
@@ -99,6 +99,13 @@ <h6 class="text-muted">{{ publisher }}</h6>
               </li>
               {% endif %}
 
+              <li class="nav-item">
```

Comment:
Since there won't immediately be data here and we don't yet have a way to backfill this data, should we hide this report during a rollout process?

Reply:
I've made it staff protected for now 👍